modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/test/testJS/select_field.js

```diff
@@ -35,18 +49,75 @@ describe(
 			}
 		);
 
-		beforeEach(
-			function(done) {
-				selectField = new Liferay.DDM.Field.Select(
-					{
-						context: {
-							name: 'selectField',
-							required: true
-						}
+		describe(
+			'.showErrorMessage()',
+			function() {
+				it(
+					'should exists an error container if the select field has an error message',
+					function(done) {
+						selectField = createSelectField();
+
+						selectField.set('errorMessage', 'error');
+
+						selectField.set('options', [{label: 'a', value: 'a'}]);
+
+						selectField.showErrorMessage();
+
+						var container = selectField.get('container');
+
+						assert.isNotNull(
```

Comment:
Maybe it's safer to test for: assert.isTrue(container.one('.help-block').inDoc())